qiskit_ibm_runtime/base_runtime_job.py

```diff
@@ -239,15 +240,12 @@ def _set_status(self, job_response: Dict) -> None:
         """
         try:
             reason = job_response["state"].get("reason")
-            reason_code = job_response["state"].get("reason_code")
+            reason_code = job_response["state"].get("reasonCode")
```

Comment:
Was it getting anything before?

Reply:
I'm not sure why/how this was changed on the server side, checking with the runtime team

Reply:
Turns out the IQP channel returns `reasonCode` while then cloud/paygo channel returns `reason_code`. Going to hold off on merging this for now because handling these inconsistencies on the client side can get messy - following up with IQP.

Reply:
Going to just handle both cases on the client side